packages/office-ui-fabric-react/src/components/SelectedItemsList/SelectedPeopleList/SelectedPeopleList.tsx

```diff
@@ -69,7 +69,7 @@ export class SelectedPeopleList extends BasePeopleSelectedItemsList {
     const props = {
       item,
       index,
-      key: item.key ? item.key : index,
+      key: item.key ? item.key + index : index,
```

Comment:
If there is a key passed in that should be left as is I think, we expect them to be unique I think